contracts/CommitManagerV1.sol

```diff
@@ -0,0 +1,329 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.4;
+
+import {Hub} from "./Hub.sol";
+import {ScoringProxy} from "./ScoringProxy.sol";
+import {ServiceAgreementV1} from "./ServiceAgreementV1.sol";
+import {Staking} from "./Staking.sol";
+import {IdentityStorage} from "./storage/IdentityStorage.sol";
+import {ParametersStorage} from "./storage/ParametersStorage.sol";
+import {ProfileStorage} from "./storage/ProfileStorage.sol";
+import {ServiceAgreementStorageV1} from "./storage/ServiceAgreementStorageV1.sol";
+import {ShardingTableStorage} from "./storage/ShardingTableStorage.sol";
+import {StakingStorage} from "./storage/StakingStorage.sol";
+import {Named} from "./interface/Named.sol";
+import {Versioned} from "./interface/Versioned.sol";
+import {ServiceAgreementStructsV1} from "./structs/ServiceAgreementStructsV1.sol";
+import {GeneralErrors} from "./errors/GeneralErrors.sol";
+import {ServiceAgreementErrorsV1} from "./errors/ServiceAgreementErrorsV1.sol";
+
+contract CommitManagerV1 is Named, Versioned {
+    event CommitSubmitted(
+        address indexed assetContract,
+        uint256 indexed tokenId,
+        bytes keyword,
+        uint8 hashFunctionId,
+        uint16 epoch,
+        uint72 indexed identityId,
+        uint40 score
+    );
+    event Logger(bool value, string message);
+
+    string private constant _NAME = "CommitManagerV1";
+    string private constant _VERSION = "1.0.0";
+
+    bool[4] public reqs = [false, false, false, false];
```

Comment:
If we have this reqs set to true on the network currently, does it make sense to have it also initially set to true now?

Reply:
I think that this should be false by default, since by default we want to perform the check.